console-boot-logs/starlink-bootlog-1.txt

```diff
@@ -0,0 +1,159 @@
+INFO:    AUTOSTARTUP MODE = 1  
+NOTICE:  Booting Trusted Firmware
+NOTICE:  BL1: v1.3(release):f26889a
+NOTICE:  BL1: Built : 13:04:48, Jul 30 2018
+INFO:    BL1: RAM 0x30200000 - 0x30209000
+INFO:    Using crypto library 'STM ATF Mini Certificates Crypto Library'
+INFO:    BL1: Searching FIP image
+INFO:    search_fip start,  slot: 0
+INFO:    search_fip end,  slot: 0, result: 0
+INFO:    VALID code at slot 0 ref_ctr 0x5555 (21845)
+INFO:    search_fip start,  slot: 1
+INFO:    search_fip end,  slot: 1, result: 0
+INFO:    VALID code at slot 1 ref_ctr 0x5555 (21845)
+INFO:    search_fip start,  slot: 2
+INFO:    search_fip end,  slot: 2, result: 0
+INFO:    VALID code at slot 2 ref_ctr 0x5555 (21845)
+INFO:    search_fip start,  slot: 3
+INFO:    search_fip end,  slot: 3, result: 0
+INFO:    VALID code at slot 3 ref_ctr 0x5555 (21845)
+INFO:    search_fip start,  slot: 4
+INFO:    search_fip end,  slot: 4, result: 0
+INFO:    TERM code at slot 4
+INFO:    Chosen slot = 3
+INFO:    BL1: Get the image descriptor
+INFO:    BL1: Loading BL2
+INFO:    Loading image id=6 at address 0x30209000
+INFO:    Skip reserving region [base = 0x30209000, size = 0x90]
+INFO:    Image id=6 loaded at address 0x30209000, size = 0x90
+INFO:    cert_nv_ctr : 1
+INFO:    plat_nv_ctr : 0
+INFO:    Loading image id=1 at address 0x30209000
+INFO:    Image id=1 loaded at address 0x30209000, size = 0xf178
+NOTICE:  BL1: Booting BL2
+INFO:    Entry point address = 0x30209000
+NOTICE:  BL20: v1.3(debug):25afa823
+NOTICE:  BL20: Built : 16:55:25, Jul 17 2020
+NOTICE:  EMMC boot counter is 47
+NOTICE:  BL2: v1.3(debug):4ac84b3
+NOTICE:  BL2: Built : 15:39:21, Oct 15 2020
+NOTICE:  Evaluate 0x8110010 & 0x10 == 0x0 -> 0
+NOTICE:  Using default targetpack config
+Boot firmware v20190415-1 starting
+!*:#)$<@[|{!!("!%5!#!!"O-#A!!!"
+Boot firmware exiting normally
+NOTICE:  BL2: end TP
+NNOTICE:  BL31: v1.3(debug):4ac84b3
+NOTICE:  BL31: Built : 15:39:24, Oct 15 2020
+
+
+U-Boot 2016.05-00307-g16c388c (Jul 23 2020 - 22:19:05 +0000)
+
+DRAM:  1004 MiB
+MMC:   Fast boot:eMMC: 8xbit - div2
```

Comment:
SpaceX used a eMMC, I feel like an eMMC dump would also lead to a good entrypoint if uBoot gives no leads. It's a bit of a mess to get going from what I've heard, and [this](https://www.riverloopsecurity.com/blog/2020/03/hw-101-emmc/) is how somebody dumped the eMMC from an Amazon Echo.